src/Features/CSharpTest/InlineTemporary/InlineTemporaryTests.cs

```diff
@@ -253,16 +252,23 @@ public void M()
         [Fact]
         public async Task Conversion_NoConversion()
         {
-            await TestFixOneAsync(
+            await TestAsync(
                 """
-                { int [||]x = 3;
+                class C
+                {
+                    void F(){ int [||]x = 3;
 
                 x.ToString(); }
+                }
+                """,
+                """
+                class C
+                {
+                    void F(){
+                        3.ToString(); }
+                }
                 """,
-                       """
-                       { 
-                       3.ToString(); }
-                       """);
+                CSharpParseOptions.Default);
```

Comment:
this test was impossible to understand in its original form (due to how it concats strings).  The updated test shows the full code sample, and shows the desired behavior (we don't touch the `{` on the method when inlining here.